wordpress-mobile/WordPress-Android
# App Data Coverage: Add tracking in WPWebView

Parent #16343 

This PR adds tracking to the following actions of WPWebViewActivity:

- Displayed
- Dismissed
- Open in external browser tapped
- Reload tapped
- Navigated back
- Navigated forward
- Share button tapped
- Device preview changed

**To test:**
- Launch app and login in (if needed)
- Navigate to My Site tab
- Tap on the site URL in the Site Header to launch the web view
- Verify the logs contain: 🔵 Tracked: webview_displayed
- Tap the reload icon in the upper right hand corner of the view
- Verify the logs contain:  🔵 Tracked: webview_reload_tapped
- Tap the Share icon on the bottom navigation bar
- Verify the logs contain:  🔵 Tracked: webview_share_tapped
- Tap the Globe icon on the bottom navigation bar
- Verify the logs contain:  🔵 Tracked: webview_open_in_browser_tapped
- Navigate back into the app from viewing in the external browser
- Tap the Device icon on the bottom navigation bar and select a different device type
-  Verify the logs contain:  🔵 Tracked: webview_preview_device_changed, Properties: {"selected_option_name":"VALUE"}
- Navigate to a page within your site within the webview to enable to back/forward navigation button on the bottom bar
- Tap the Navigate back button
- Verify the logs contain: 🔵 Tracked: webview_navigated_back
-  Tap the Navigation forward button
- Verify the logs contain: 🔵 Tracked: webview_navigated_forward
- Return the My Site tab using the back arrow or back button
-  Verify the logs contain: 🔵 Tracked: webview_dismissed
## Regression Notes
1. Potential unintended areas of impact
Only added track events

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual Testing

3. What automated tests I added (or what prevented me from doing so)
Updated WPWebViewViewModelTest to account for tracker

PR submission checklist:
